pkg/crc/machine/bundle/metadata.go

```diff
@@ -217,3 +219,40 @@ func GetCustomBundleName(bundleFilename string) string {
 	baseName := re.ReplaceAllLiteralString(bundleFilename, "")
 	return fmt.Sprintf("%s_%d%s", baseName, time.Now().Unix(), bundleExtension)
 }
+
+type Bundle struct {
+	Openshift struct {
+		Amd64 struct {
+			Linux struct {
+				URL    string `json:"url"`
+				Sha256 string `json:"sha256"`
+			} `json:"linux"`
+			Darwin struct {
+				URL    string `json:"url"`
+				Sha256 string `json:"sha256"`
+			} `json:"darwin"`
+			Windows struct {
+				URL    string `json:"url"`
+				Sha256 string `json:"sha256"`
+			} `json:"windows"`
+		} `json:"amd64"`
+	} `json:"openshift"`
+}
+
+func GetBundleURLAndSha256sum(data []byte) (string, string, error) {
```

Comment:
With the comment above, this can become `GetBundleDownloadDetails()` to avoid exposing url and sha256sum too much.
`data []byte` in the args is not very nice. What should I pass to it? I'd expect to either be able to pass it the parsed data, or to not pass it any args, and it would be doing everything by itself.